alephmarcreader/abstractalephmarcreader.py

```diff
@@ -109,6 +109,67 @@ def get_pretty_string(self):
             else:
                 return self.reference
 
+    class StandardizedDate:
+
+        CALENDAR_PREFIX_KNORA = u'GREGORIAN:'
```

Comment:
CALENDAR_PREFIX_KNORA = u'GREGORIAN' + TIMESPAN_SEPARATOR_KNORA

Reply:
you're right, of course